nc/views/likelihood.py

```diff
@@ -31,28 +31,43 @@ def qs(self):
 def likelihood_stop_query(request, agency_id, debug=False):
     """
     Query LikelihoodStopSummary view for stop likelihood data.
-
-    Related notebooks:
-    - https://nccopwatch-share.s3.amazonaws.com/2024-04-likelihood-of-stops/likelihood-of-stops.html
```

Comment:
I find the notebook helpful context. Any reason why we'd delete it?

Reply:
I deleted that in error -- happy to add it back. Thanks for catching that!